client/extensions/wp-super-cache/components/navigation/index.jsx

```diff
@@ -13,85 +15,46 @@ import SectionNav from 'components/section-nav';
 import SectionNavTabs from 'components/section-nav/tabs';
 import SectionNavTabItem from 'components/section-nav/item';
 import { addSiteFragment } from 'lib/route/path';
+import { getSiteSlug } from 'state/sites/selectors';
 import { Tabs } from '../../app/constants';
 
-const Navigation = ( { activeTab, site, translate } ) => {
-	const getLabel = tab => {
-		switch ( tab ) {
-			case Tabs.EASY:
-				return translate( 'Easy' );
-			case Tabs.ADVANCED:
-				return translate( 'Advanced' );
-			case Tabs.CDN:
-				return translate( 'CDN' );
-			case Tabs.CONTENTS:
-				return translate( 'Contents' );
-			case Tabs.PRELOAD:
-				return translate( 'Preload' );
-			case Tabs.DEBUG:
-				return translate( 'Debug' );
-		}
-	};
-
-	const getTabs = () => {
-		const tabs = [];
-
-		for ( const key in Tabs ) {
-			if ( Tabs.hasOwnProperty( key ) ) {
-				tabs.push( Tabs[ key ] );
-			}
-		}
-
-		return tabs;
-	};
-
-	const renderTabItems = ( tabs ) => {
-		return tabs.map( tab => {
-			let path = '/extensions/wp-super-cache';
-
-			if ( tab !== Tabs.EASY ) {
-				path = `${ path }/${ tab }`;
-			}
-
-			if ( site ) {
-				path += `/${ site.slug }`;
-			}
-
-			return (
-				<SectionNavTabItem
-					key={ `wp-super-cache-${ tab }` }
-					path={ path }
-					selected={ ( activeTab || Tabs.EASY ) === tab }>
-					{ getLabel( tab ) }
-				</SectionNavTabItem>
-			);
-		} );
-	};
-
-	const pluginUrl = addSiteFragment( '/plugins/wp-super-cache', get( site, 'slug' ) );
-	return (
-		<div>
-			<HeaderCake backText={ translate( 'Plugin Overview' ) }
-				backHref={ pluginUrl }>
-				WP Super Cache
-			</HeaderCake>
-			<SectionNav selectedText="Settings">
-				<SectionNavTabs>
-					{ renderTabItems( getTabs() ) }
-				</SectionNavTabs>
-			</SectionNav>
-		</div>
-	);
-};
+const Navigation = ( { activeTab, siteSlug, translate } ) => (
+	<div>
+		<HeaderCake backText={ translate( 'Plugin Overview' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **88** times:
`translate( 'Overview' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/wpsc-cleanup)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).